test/language/expressions/arrow-function/arrow/apply_0.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: invoke arrow function using apply
+es6id: 14.2
+---*/
+
+var a = (x) => x;
+
+assert.sameValue(typeof a.apply, "function", "Apply is not a member exression on arrow function (x) => x.");

```

Comment:
The apply property comes from the prototype of an arrow function object, which should be Function.prototype.  Instead of testing for existence of the apply property it would make more sense to test that the prototype objet is correct and assume prototype inheritance semantics are correct such that apply is invokable off an arrow function.

It is still good to test calling an arrow function with apply so the tests below are good to keep.